app/database/controller.py

```diff
@@ -1,54 +1,66 @@
 from fastapi import FastAPI,APIRouter
-from app.database.schema import client, Message, db
-from bson.objectid import ObjectId
-some_router = APIRouter() 
+from datetime import datetime
+import dateutil.parser
+from app.database.schema import client, Message, db 
+from app.database.body import Messagebody
+from bson import ObjectId 
+import datetime 
+from tzlocal import get_localzone # $ pip install tzlocal
 
-def get_message():
+
+some_router = APIRouter()
+
+
+def get_message(channel_id,starttime,endtime):
     """
     Args : None 
     
     The Function is used to fetch all the data in the DB 
     
     Return Type : List 
-    
+    {"channel_id":channel_id })
     """
+    starttime = dateutil.parser.parse(starttime)
+    endtime = dateutil.parser.parse(endtime)
+    print(starttime)
+    print(endtime)
     cynergy = []
-    print (db.message.find())
```

Comment:
create separate file containing DB methods and call appropriate method whenever needed. Instead of writing `db.message.find()` you should write `message_db().find()`. Do this everywhere!